### ISSUE TITLE
Multiple entries with url_resolved "http://streaming.radionomy.com/JamendoLounge"

### ISSUE THREAD

**Issue by smoralis** [segler-alex/radiobrowser#219]
Different urls / stations have the same url_resolved http://streaming.radionomy.com/JamendoLounge

[JamendoLounge.txt](https://github.com/segler-alex/radiobrowser/files/4971069/JamendoLounge.txt)

**Comment by wolfmanfp** [comment]
Maybe this is caused by the shutdown of the free Radionomy service: http://www.radionomy.com/index.html

**Comment by smoralis** [comment]
Whatever the cause , validation should be used to keep url & url_resolved unique and point to one station.

**Comment by wolfmanfp** [comment]
You're right, but that was not a validation error. These stations were hosted by Radionomy, and were added to the database long before they shut down the service and redirected all streams. You can't avoid things like this.
